get-started/quickstart.rst

```diff
@@ -0,0 +1,383 @@
+============
+
+Zeek is often run on live traffic on a network, not just captured traffic.
+You can provide Zeek with a network interface to monitor traffic. Any traffic on
+that interface will be analyzed in order to create logs. For example, you may
+run Zeek on the ``en0`` network device (change ``en0`` to the device you want
+to monitor traffic on):
+
+.. code-block:: console
+
+     $ zeek -i en0
+
+Then, in another terminal, create the same two HTTP requests we saw earlier via
+``curl``:
+
+.. code-block:: console
+
+     $ curl -X GET http://zeek.org
+     $ curl -X WEIRD http://zeek.org
+
+Zeek will analyze traffic over that network device. Therefore, the logs may
+have more than just the two entries found before. Entries should still appear in
+:file:`conn.log`, :file:`http.log`, and :file:`weird.log` for these commands.
+
+Scripting
+=========
+
+Zeek can also use its own scripting language in order to configure behavior and
+react to events:
+
+.. code-block:: zeek
+
+     event http_request(c: connection, method: string, original_URI: string,
+         unescaped_URI: string, version: string)
+         {
+         print fmt("HTTP request: %s %s (%s->%s)", method, original_URI, c$id$orig_h,
+             c$id$resp_h);
+         }
+
+This script defines an event that will run whenever Zeek sees an HTTP request.
+You can run it through Zeek with the data from the pcap:
+
+.. code-block:: console
+
+     $ zeek weird-test.zeek -r quickstart.pcap
+     HTTP request: GET / (192.168.1.8->192.0.78.212)
+     HTTP request: WEIRD / (192.168.1.8->192.0.78.212)
+
+Or on live traffic:
+
+.. code-block:: console
+
+     $ zeek -i en0
+
+In another terminal, run the two ``curl`` commands from before:
+
+.. code-block:: console
+
+     $ curl -X GET http://zeek.org
+     $ curl -X WEIRD http://zeek.org
+
+The terminal running Zeek will print each command as it gets processed.
+
+More information on how to use Zeek’s scripting language can be found in the
+:doc:`../scripting/index` section.
+
+Managing Zeek
+=============
+
+Zeek comes packaged with ZeekControl (``zeekctl``) to manage more complex use
+cases.
+
+The same network device used in the Zeek command line can be used with
+``zeekctl``. This will go in a configuration file. For the following example,
+``$PREFIX`` will refer to the installation directory. This is likely
+``/usr/local/zeek`` if built from source or ``/opt/zeek`` if installed from a
+pre-built package.
+
+First, update the configuration’s interface in ``$PREFIX/etc/node.cfg``. If
+the device is ``en0``, that would look like:
+
+.. code-block:: console
+
+     [zeek]
+     type=standalone
+     host=localhost
+     interface=en0
+
+You can also further configure the ``local.zeek`` script found in
+``$PREFIX/share/zeek/site/local.zeek``. This script is loaded by ``zeekctl`` by
+default. It is not overwritten by Zeek upgrades. Zeek without ``zeekctl`` may
+specify ``local.zeek`` in the command line in order to load its contents.
```

Comment:
While `local.zeek` not being added to a bare `zeek` invocation is definitely a known gotcha, unclear sneaking it in here is useful - maybe leave it out?

Reply:
you're right - I think just waiting until this point to mention it might be enough context without adding an extra explanation